modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -746,6 +771,12 @@ Sample usage of detecting and drawing the centers of circles: :
 the board to make the detection more robust in various environments.
  */
 CV_EXPORTS_W bool findCirclesGrid( InputArray image, Size patternSize,
+                                   OutputArray centers, int flags,
+                                   const Ptr<FeatureDetector> &blobDetector,
+                                   CirclesGridFinderParameters parameters);
+
+/** @overload */
```

Comment:
Please also add here a small explanation why you are using the overload method and how the extra parameters should be used. An example can bee seen at the detectMultiScale functionality:

http://docs.opencv.org/master/da/dd5/classcv_1_1BaseCascadeClassifier.html#afa5ead715ec7c9cf0ef71f210ba7715f